### PR TITLE
feat: sweep alby shared funds balance

### PR DESCRIPTION
**Pull request by rolznz**
Allows a user to send the rest of their funds from the Alby shared node to their own node if they have enough receiving capacity. Otherwise, they will be guided to increase their receiving capacity.

Closes https://github.com/getAlby/nostr-wallet-connect-next/issues/383

### CODE REVIEW

File: frontend/src/screens/channels/Channels.tsx [hunk 1]
```diff
@@ -348,6 +361,52 @@ export default function Channels() {
...
+                    await request("/api/alby/drain", {
+                      method: "POST",
+                      headers: {
+                        "X-CSRF-Token": csrf,
+                        "Content-Type": "application/json",
+                      },
+                    });
+                    await reloadAlbyBalance();
+                    toast({
+                      description: "🎉 Funds swept from Alby shared wallet!",
```
Comment (bumi) [inline frontend/src/screens/channels/Channels.tsx:394]: is "funds swept" a common term?

File: alby/alby_oauth_service.go [hunk 2]
```diff
@@ -247,6 +249,39 @@ func (svc *albyOAuthService) GetBalance(ctx context.Context) (*AlbyBalance, erro
...
+	if err != nil {
+		logger.Logger.WithError(err).Error("Failed to fetch shared balance")
+		return err
+	}
+
+	amount := int64(math.Floor(
+		float64(balance.Balance)*1000* // Alby shared node balance in sats
+			(1-8/1000)* // Alby service fee (0.8%)
+			0.99)) - // Maximum potential routing fees (1%)
+		10000 // Alby fee reserve (10 sats)
```
[the remark this answers is not in the document]
Comment (rolznz) [reply alby/alby_oauth_service.go:263]: it should be maximum of 1% or 10 sats, shouldn't it? which are both subtracted from the balance here on the safe side, to ensure the payment is successful

File: frontend/src/screens/channels/Channels.tsx (hunk 1 above)
Comment (rolznz) [reply frontend/src/screens/channels/Channels.tsx:394]: @reneaaron mentioned the same thing. He suggested "Migrate" or maybe "Take Custody" :raised_hands: 

Any thoughts?

File: alby/alby_oauth_service.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (bumi) [reply alby/alby_oauth_service.go:263]: the minimum is 10 sats: https://github.com/getAlby/lndhub.go/blob/fc4fe444ad62374f46aaa78dd0edaf6b218c487d/lib/service/user.go#L288-L295

[the remark this answers is not in the document]
Comment (rolznz) [reply alby/alby_oauth_service.go:263]: @bumi this is part of the calculation - I subtract 1% and then subtract 10 sats, and only continue if the amount to migrate is bigger than 0. It should work, although a few sats might be left over, but at least 10 sats will be left over anyway, and if there's still more than 100 sats the user can migrate one more time to get the rest.

File: frontend/src/screens/channels/Channels.tsx (hunk 1 above)
Comment (rolznz) [reply frontend/src/screens/channels/Channels.tsx]: I updated to "Take Custody"

**Pull request merged**